react/src/components/Shoe.js

```diff
@@ -3,10 +3,12 @@ import React from 'react';
 const Shoe = (props) => {
   return(
     <div>
-      <h2><a href={'/shoes/' + props.id}>{props.brand} {props.model} {props.version}</a></h2>
-      <img src={`${props.picture}`}/>
-      <h4>{props.description} {props.year}</h4>
-      <p>{props.price}</p>
+      <div className="row">
+        <div className="shoe">
+        <h2><a href={'/shoes/' + props.id}>{props.brand} {props.model} {props.version}</a></h2>
+        <img src={`${props.picture}`}/>
```

Comment:
This doesn't seem to be wrong. Also we have it ignoring react so not sure why it went in here. Let us know if it is a problem!